code/game/gamemodes/wizard/magic_tarot.dm

```diff
@@ -193,6 +195,8 @@
 
 /obj/item/magic_tarot_card/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
 	. = ..()
+	if(needs_mob_target && !isliving(hit_atom))
```

Comment:
why use a variable if it never changes?
```suggestion
	if(!isliving(hit_atom))
```

Reply:
It's there if you want to add a new card  that doesn't require a mob target